File: hw/chavez/u3/hw21AnimalFarm/python/model/FarmAnimal.py
class FarmAnimal:

    def __init__(self, id, breed, born_on):
        self.id = id
        self.breed = breed
        self.born_on = born_on

    def feed(self, food):
        print(f"Feeding the {self.__class__.__name__.upper()} with food -> {food.name} in a bowl and water")

File: hw/chavez/u3/hw21AnimalFarm/javascript/model/FarmAnimal.js
export default class FarmAnimal {
    constructor(id, breed, bornOn) {
        this.id = id;
        this.breed = breed;
        this.bornOn = bornOn;
    }

    feed(food) {
        console.log(`Feeding the ${this.constructor.name.toUpperCase()} with food -> ${food.name} in a bowl and water`);
    }
}